tornado/platform/asyncio.py

```diff
@@ -576,7 +576,21 @@ def _run_select(self) -> None:
                         raise
                 else:
                     raise
-            self._real_loop.call_soon_threadsafe(self._handle_select, rs, ws)
+
+            try:
+                self._real_loop.call_soon_threadsafe(self._handle_select, rs, ws)
+            except RuntimeError:
+                # "Event loop is closed". Swallow the exception for
+                # consistency with PollIOLoop (and logical consistency
+                # with the fact that we can't guarantee that an
+                # add_callback that completes without error will
+                # eventually execute).
+                pass
+            except AttributeError:
```

Comment:
FYI this was fixed in I believe python 3.8.5: https://bugs.python.org/issue39651. I'm fine keeping this around for compatibility though.